code/game/objects/items/reagent_containers/food/snacks/meat.dm

```diff
@@ -57,9 +57,19 @@
 
 /obj/item/reagent_container/food/snacks/meat/xenomeat/Initialize()
 	. = ..()
-	reagents.add_reagent("xenoblood", 3)
+	reagents.add_reagent("xenoblood", 6)
 	src.bitesize = 6
 
+/obj/item/reagent_container/food/snacks/meat/xenomeat/processed
+	name = "meat"
```

Comment:
Just to check: did you want to change the name of this to "processed meat" like on the carp fillet, or have only the description be different?

Reply:
I wasn't sure of an elegant way to have it called processed NAME meat (e.g. processed Queen meat) so opted to just have it as NAME meat, which is handled in process. To my ears, "NAME processed meat" feels off but happy to take that approach if it'd be better.

Reply:
Hm.. It could be done by refactoring how the `made_from_player` variable is handled, but that might be out of the scope of the PR.
Either way, this works fine as it is now IMO. 👍